lib/error-helper.spec.js

```diff
@@ -1,52 +1,49 @@
 'use strict'
 
-const chai = require('chai')
-const { assert, expect } = chai
+const { expect } = require('chai')
 const { checkErrorResponse } = require('./error-helper')
 const { NotFound, InvalidResponse } = require('../services/errors')
 
-chai.use(require('chai-as-promised'))
-
 describe('Standard Error Handler', function() {
   it('makes inaccessible badge', function() {
     const badgeData = { text: [] }
-    assert.equal(
-      true,
-      checkErrorResponse(badgeData, 'something other than null', {})
-    )
-    assert.equal('inaccessible', badgeData.text[1])
-    assert.equal('red', badgeData.colorscheme)
+    expect(checkErrorResponse(badgeData, 'something other than null', {})).to.be
+      .true
```

Comment:
Is this on a new line because of prettier/eslint?
I feel like it would look better on 1 line, but i'm fine with it either way.

Reply:
Yes, I believe this is due to Prettier.